examples/python-poetry/devenv.nix

```diff
@@ -15,6 +15,23 @@
 
   languages.python = {
     enable = true;
-    poetry.enable = true;
+    poetry = {
+      enable = true;
+      install = {
+        enable = true;
+        installRootPackage = false;
+        onlyInstallRootPackage = false;
+        compile = false;
+        quiet = false;
+        groups = [ ];
+        ignoredGroups = [ ];
+        onlyGroups = [ ];
+        extras = [ ];
+        allExtras = false;
+        verbosity = "no";
+      };
+      activate.enable = true;
+      package = pkgs.poetry;
```

Comment:
By default people shouldn't need to set all of these options, should the test check for options or for the default poetry environment? :sweat_smile: